cicd/3-app/javabuilder/config/adhoc.config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Parameters": {
+    "BaseDomainName": "code.org",
+    "BaseDomainNameHostedZonedID": "Z2LCOI49SCXUGU",
+    "ProvisionedConcurrentExecutions": "1",
+    "ReservedConcurrentExecutions": "3",
+    "LimitPerHour": "15",
```

Comment:
can we set the hourly limit to 50 and the daily limit to 150? For dev instances we'll want > 15 usages per hour